source/MaterialXGraphEditor/External/Glfw/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ if (BUILD_SHARED_LIBS AND UNIX)
     # On Unix-like systems, shared libraries can use the soname system.
     set(GLFW_LIB_NAME glfw)
 else()
-    set(GLFW_LIB_NAME glfw3)
+    set(GLFW_LIB_NAME glfw3_gred)
```

Comment:
This looks like a good catch, thanks @igor-elovikov.  To align this new name with similar ones that distinguish the version of GLFW in the MaterialX Graph Editor, could we name this library `glfw3_minimal`?